time timestamp
float64[] id
int64[] occupiedPosx
int64[] occupiedPosy
int64[] occupiedPosx1
int64[] occupiedPosy1
float64[] pos
float64[] rectifypos
float64[] posEstMean
float64[] particles
int64 state

